Makefile

```diff
@@ -139,10 +139,10 @@ docker-push: ## Push docker image with the manager.
 ##@ Deployment
 
 install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+	$(KUSTOMIZE) build config/crd | kubectl replace -f -
```

Comment:
we  need to find a solution for this . this is kubectl issue. when we run apply, it copies the whole CR and adds that to annotations as history, but since our CR is too big , its more than 1MB the limitation from etcd database for any row data. so kubectl fails. there is already an issue raised by many on this. replace is not a good idea as this will delete the CRD and replaces that with latest. but `make install` command is used only by developer it should be ok. we need to find a way to fix this for user

Reply:
Rolled back in the latest commit.